low-level/sensors/impl/platforms/realstor/realstor_fan_sensor.py

```diff
@@ -48,7 +48,7 @@ class RealStorFanSensor(SensorThread, InternalMsgQ):
 
     SENSOR_NAME = "RealStorFanSensor"
     SENSOR_TYPE = "enclosure_fan_module_alert"
-    RESOURCE_TYPE = "enclosure:hw:fan"
+    RESOURCE_TYPE = "storage:hw:fan"

```

Comment:
I see enclosure actuator request type as `ENCL:storage:hw:----`, why here it's not `ENCL:storage:hw:fan` ??

Reply:
or is `ENCL` prefix only for actuator requests??

Reply:
yes ENCL added for requests